src/app/features/notification/model/notification-config.ts

```diff
@@ -0,0 +1,28 @@
+import { Entity } from "../../../core/entity/model/entity";
+import { DatabaseField } from "../../../core/entity/database-field.decorator";
+import { DatabaseEntity } from "../../../core/entity/database-entity.decorator";
+
+/**
+ * This represents one specific notification config for one specific user,
+ */
+@DatabaseEntity("NotificationConfig")
+export class NotificationConfig extends Entity {
+  @DatabaseField() channels: { [key: string]: boolean };
```

Comment:
```suggestion
  @DatabaseField() channels: { [key: string]: boolean };
```

Can we introduce a `type NotificationChannel = "push"; ` instead of allowing any string for the keys? (later to add other options beyond push, was implemented.